api/controllers/AuthController.js

```diff
@@ -110,13 +119,16 @@ module.exports = {
     }
 
     params = params.asObject();
+    var finduser;
 
     if (authConfig.wetland) {
       UserEntity = sails.models.user.Entity;
       manager    = req.getManager();
       findUser   = manager.getRepository(UserEntity).findOne({[loginProperty]: params[loginProperty]});
     } else {
-      findUser = sails.models.user['findOneBy' + _.upperFirst(loginProperty)](params[loginProperty]);
+      sails.models.user.findOne({username: 'test'}).exec(function (err, user){
```

Comment:
`findUser = sails.models.user.findOne({[loginProperty]: params[loginProperty]});`

Reply:
Undo the exec logic